lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -196,7 +191,8 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     @Input()
     set showFilesInResult(value: boolean) {
         if (value !== undefined && value !== null) {
-            this.showFiles = value;
+            const showFilesQuery = `TYPE:'cm:folder'${value ? " OR TYPE:'cm:content'" : ''}`;
```

Comment:
this can be done via app.config, we have support for providing this values

Reply:
check this example: https://github.com/Alfresco/alfresco-content-app/blob/develop/src/app.config.json#L265